sw/device/lib/testing/aes_testutils.c

```diff
@@ -19,7 +19,7 @@
 #include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
 #endif
 
-#define TIMEOUT (1000 * 1000)
+#define TIMEOUT (10 * 1000 * 1000)
```

Comment:
Can you please define an enum for this constant as well? Just like you did for the other cases.

Reply:
Sure!